docs/installation.rst

```diff
@@ -76,6 +79,22 @@ a. Set up ``udev`` rules for appropiate permission to ``/dev/ttyACM*`` in Linux
 
     1. Create a file named ``setup_udev_rule.sh`` and include the following script
 
+Mac
+^^^
+1. Install ``XCode`` from the Mac App store. For this, you may need to upgrade to the latest version of MacOS. For older versions of MacOS, find compatible versions of ``XCode`` `here <https://en.wikipedia.org/wiki/Xcode>`_. All old ``XCode`` versions are available `here <https://developer.apple.com/download/more/>`_.
+2. Accept the license agreement: ``sudo xcodebuild -license``
+3. It is best to install Anaconda. Download  and install the `Anaconda Python 3.7 Mac installer <https://www.anaconda.com/distribution/#download-section>`_. For older versions of MacOS, compatible version of Anaconda can be found in `this table <https://docs.continuum.io/anaconda/install/#old-os>`_ and downloaded `here <https://repo.anaconda.com/archive/index.html>`_.
+4. We recommend using a virtual environment in Conda.
+
+   a. In Conda command prompt: ``conda create -n myenv python=3.10``
+   b. Activate the virtual environment: ``conda activate myenv``
+
+5. Upgrade your pip: ``python -m pip install --upgrade pip``
+6. Run: ``pip install explorepy``, to install ``explorepy`` from PyPI.
+7. Run: ``brew install blueutil``, to install blueutil for bluetooth communication
```

Comment:
Homebrew doesn't come preinstalled on Mac, I think, so it should be linked somewhere for installation